### PR TITLE
Portfolio: Mobile version skeleton

### PR DESCRIPTION
**Pull request by Krishnabot**
In this project

- HTML and CSS files for work and about and contact sections created and tested. 

- github flow created.

- linter set up and checked for any linting errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,13 +14,110 @@
...
+        <li><a href="https://twitter.com/last_matrix" target="_blank" rel="noopener"><img src="src/twitter.svg" alt="Twitter Icon"></a></li>
+        <li><a href="https://www.linkedin.com/in/krishna-prasad-acharya-3596bb130/" target="_blank" rel="noopener"><img src="src/Linkedin-icon.svg" alt="Linkedin icon"></a></li>
+        <li><a href="#"><img src="src/mail.svg" alt="Mail Icon"></a></li>
+        <li><a href="https://github.com/Krishnabot" target="_blank" rel="noopener"><img src="src/github.svg" alt="GiThub icon"></a></li>
+        <li><a href="#"><img src="src/v_sign.svg" alt="V sign"></a></li>
         
-        <div>LET'S CONNECT</div>
+        </ul>
+        </div>       
+        <div class="projects">
```
Comment (jaflih) [inline index.html:33]: Kindly change the `div` by `section`, the semantic help the browser to know witch section it gonna display as well.

Comment (jaflih) [inline index.html:33]: Please kindly change your code for using `CSS grid ` with `Flexbox` to layout elements like asked in the requirement: 

image

File: index.html [hunk 2]
```diff
@@ -14,13 +14,110 @@
...
+                <li>2015</li>
+            </ul>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <div class="tags">
+                <button type="button">html</button>
+                <button type="button">css</button>
+                <button type="button">javascript</button>
+            </div>
```
Comment (jaflih) [inline index.html]: Please, kindly change your code using `ul/li `as asked in the requirement:

image

File: index.html [hunk 3]
```diff
@@ -14,13 +14,110 @@
...
```
Comment (jaflih) [inline index.html]: Please ensure your button are conform to the design guidelines by Figma:

Your design
image

Figma:
image

File: index.html [hunk 4]
```diff
@@ -14,13 +14,110 @@
...
+            <div>
+                <button type="button">See Project</button>
+            </div>
+            
+        </div>
+        <div class="about-myself">
```
Comment (jaflih) [inline index.html]: Kindly change the `div` by `section`, the semantic help the browser to know which section it gonna display as well.

File: index.html [hunk 5]
```diff
@@ -29,8 +126,50 @@ <h1 id="hello">I’m Krishna <br>
         <li><a href="#"><img src="src/v_sign.svg" alt="V sign"></a></li>
         
         </ul>
-    
+        <button type="button">Get my resume</button>
```
Comment (jaflih) [inline index.html:149]: Please ensure your button hover effects conform to the design guidelines by Figma:

image

File: index.html [hunk 6]
```diff
@@ -29,8 +126,50 @@ <h1 id="hello">I’m Krishna <br>
...
+
+        </div>
+        
+        <div class="skills">
+            <div>
+                <p>Languages</p>
```
Comment (jaflih) [inline index.html]: Please, kindly change your code (add `ul/li`) to follow the requirement:

image

File: index.html (hunk 4 above)
Comment (jaflih) [inline index.html]: Please, use `Flexbox` to layout this section.  Using Flexbox will facilitate the design when you will do the Desktop view.

File: style.css [hunk 7]
```diff
@@ -108,3 +146,203 @@ ul > li {
...
+  gap: 10px;
+  position: absolute;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  background-color: white;
+  color: #6070ff;
+}
+
+.about-myself {
+  position: relative;
```
Comment (jaflih) [inline style.css]: [OPTIONAL]Please, don't use the position: relative;`, instead use margin/padding if you want to space element. The problem is that you give a height to your section, that's why you have this problem. 

image

Please, remove it and add margin/padding to have a professional project.

**Review by jaflih — CHANGES_REQUESTED** [review]
Hi @Krishnabot  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
✔️ Linters passed 💯
✔️ The content is aligned with flexbox 🥇 
✔️ Great commit history

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @jaflih in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vic778 — APPROVED** [review]
Hi @Krishnabot  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification

**Pull request merged**